README.md

```diff
@@ -18,7 +18,10 @@ Note: You will need to create a figma account (it’s free) to download the asse
 
 5. This will continue until a user correctly guesses 5 times. They are then taken to a stats screen which displays the number of correct and incorrect guesses. The user can select the “home” button which takes them back to the splash screen.
 
-6. You can implement this app however you choose, that said, your Name Game test project will be evaluated according to [this rubric](namegame_web_evaluation_rubric.pdf). 
+6. You can implement this app however you choose, that said, your Name Game test project will be evaluated according to [this rubric](namegame_web_evaluation_rubric.pdf).
```

Comment:
Will we need to link to `namegame_web_evaluation_rubric.docx` instead here?

Reply:
Nah I updated the pdf, as well. The docx is just so it's easily editable. Not everyone will be able to open a .docx.

Reply:
Awesome, thank you! :)